src/blocks/wiki-embed/edit.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * BLOCK: Tabs
+ */
+const { InspectorControls } = wp.blockEditor;
+const { PanelBody, Button, ButtonGroup, CheckboxControl, TextControl } = wp.components;
+const ServerSideRender = wp.serverSideRender;
+const { useState } = wp.element;
+
+const Edit = ( { attributes, setAttributes } ) => {
+	const { source, headingType, noEditLinks, noTabContens, noInfoBox } = attributes;
+	const [ newSource, setNewSource ] = useState( source );
+	const headingTypeOptions = {
+		Default: '0',
+		Tabs: '1',
+		Accordions: '2',
+	};
+
+	/**
+	 * Render default gray box to insert url when source is empty.
+	 */
+	function renderDefaultView() {
+		return (
+			<div className="ubc-wiki-embed-default-view">
+				<h2 className="ubc-wiki-embed-default-view__heading">Wiki Embed, please insert url below:</h2>
```

Comment:
Let's try

"Paste URL for OpenWiki resource, i.e. https://wiki.ubc.ca/Course:Math340"

Reply:
Also, let's use the `__` package to make this - and all other text strings - translatable